src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -199,5 +209,13 @@ private void CheckIfAnonymizedExportIsEnabled()
                 throw new RequestNotValidException(string.Format(Resources.OperationNotEnabled, OperationsConstants.AnonymizedExport));
             }
         }
+
+        private static void CheckOutputFormat(string outputFormat)
```

Comment:
nit: any reason we don't want to do the validation in the `ValidateExportRequestFilterAttribute` ? Seems like a good candidate to be part of the validation we do there.

Reply:
I debated this. I didn't put it in the ValidateExportRequestFilterAttribute because so far all the validation we do in there is for header values, not parameter values.

Reply:
Moved